RecoTracker/IterativeTracking/python/HighPtTripletStep_cff.py

```diff
@@ -87,7 +87,7 @@
     minPt = 0.2,
 )
 highPtTripletStepTrajectoryFilterBase = _highPtTripletStepTrajectoryFilterBase.clone(
-    maxCCCLostHits = 2,
+    maxCCCLostHits = 0,
     minGoodStripCharge = dict(refToPSet_ = 'SiStripClusterChargeCutLoose')
```

Comment:
highPtTripletStepChi2Est is left with SiStripClusterChargeCutTiny
Is this intended?